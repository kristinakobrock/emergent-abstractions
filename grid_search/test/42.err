  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 18.68it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 22.07it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 23.29it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 29.20it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 27.53it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 27.17it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 27.93it/s] 50%|##############################################                                              | 25/50 [00:00<00:00, 31.21it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 27.66it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 25.93it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 25.69it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 24.52it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 23.82it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 25.29it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 24.41it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.84it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.96it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 19.80it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 25.89it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 22.82it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 21.58it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.30it/s]
