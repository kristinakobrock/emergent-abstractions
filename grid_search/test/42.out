# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7492156624794006, "acc": 0.584408164024353, "length": 3.505373239517212, "mode": "train", "epoch": 1}
{"loss": 0.7026271820068359, "acc": 0.4814452826976776, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6882801055908203, "acc": 0.5474109649658203, "length": 3.2584874629974365, "mode": "train", "epoch": 2}
{"loss": 0.6611400246620178, "acc": 0.6339843273162842, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.674342155456543, "acc": 0.6276448369026184, "length": 2.67215895652771, "mode": "train", "epoch": 3}
{"loss": 0.6785762906074524, "acc": 0.5847654938697815, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6658247709274292, "acc": 0.6179277896881104, "length": 1.8519890308380127, "mode": "train", "epoch": 4}
{"loss": 0.659324049949646, "acc": 0.6355469226837158, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6614683866500854, "acc": 0.6280713677406311, "length": 1.8272011280059814, "mode": "train", "epoch": 5}
{"loss": 0.6566665172576904, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6596717834472656, "acc": 0.6266784071922302, "length": 1.6187669038772583, "mode": "train", "epoch": 6}
{"loss": 0.6568800210952759, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6580938696861267, "acc": 0.6275888681411743, "length": 1.4517372846603394, "mode": "train", "epoch": 7}
{"loss": 0.6559792757034302, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6583236455917358, "acc": 0.6277554035186768, "length": 1.3724911212921143, "mode": "train", "epoch": 8}
{"loss": 0.6560949683189392, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6567308306694031, "acc": 0.6275770664215088, "length": 1.3578542470932007, "mode": "train", "epoch": 9}
{"loss": 0.6558301448822021, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6565364599227905, "acc": 0.6282036304473877, "length": 1.3372529745101929, "mode": "train", "epoch": 10}
{"loss": 0.6564235687255859, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6539350152015686, "acc": 0.6285402774810791, "length": 1.4898467063903809, "mode": "train", "epoch": 11}
{"loss": 0.6572715044021606, "acc": 0.63671875, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6492854952812195, "acc": 0.6313833594322205, "length": 1.8483288288116455, "mode": "train", "epoch": 12}
{"loss": 0.6528028845787048, "acc": 0.6423202753067017, "length": 1.0703125, "mode": "test", "epoch": 12}
{"loss": 0.6381756663322449, "acc": 0.6436585187911987, "length": 1.9128947257995605, "mode": "train", "epoch": 13}
{"loss": 0.631312370300293, "acc": 0.6563934683799744, "length": 1.6796875, "mode": "test", "epoch": 13}
{"loss": 0.6246206164360046, "acc": 0.6585196256637573, "length": 2.1436638832092285, "mode": "train", "epoch": 14}
{"loss": 0.6158236265182495, "acc": 0.6719436645507812, "length": 2.18359375, "mode": "test", "epoch": 14}
{"loss": 0.6139014363288879, "acc": 0.6705383062362671, "length": 2.170701026916504, "mode": "train", "epoch": 15}
{"loss": 0.6098111867904663, "acc": 0.6823500990867615, "length": 2.25390625, "mode": "test", "epoch": 15}
{"loss": 0.606408953666687, "acc": 0.6786865592002869, "length": 2.1592345237731934, "mode": "train", "epoch": 16}
{"loss": 0.6063574552536011, "acc": 0.678301990032196, "length": 2.20703125, "mode": "test", "epoch": 16}
{"loss": 0.5991796851158142, "acc": 0.6837915778160095, "length": 2.14211368560791, "mode": "train", "epoch": 17}
{"loss": 0.6005097031593323, "acc": 0.6726287603378296, "length": 2.0078125, "mode": "test", "epoch": 17}
{"loss": 0.5920303463935852, "acc": 0.6872701048851013, "length": 2.040628671646118, "mode": "train", "epoch": 18}
{"loss": 0.5984909534454346, "acc": 0.6891494989395142, "length": 2.23046875, "mode": "test", "epoch": 18}
{"loss": 0.5862007141113281, "acc": 0.6933966875076294, "length": 2.1140782833099365, "mode": "train", "epoch": 19}
{"loss": 0.6011402606964111, "acc": 0.6758270263671875, "length": 2.171875, "mode": "test", "epoch": 19}
{"loss": 0.5798018574714661, "acc": 0.6989142894744873, "length": 2.2061426639556885, "mode": "train", "epoch": 20}
{"loss": 0.5919229984283447, "acc": 0.6894500255584717, "length": 2.27734375, "mode": "test", "epoch": 20}
{"loss": 0.5656474232673645, "acc": 0.7077054381370544, "length": 2.2810230255126953, "mode": "train", "epoch": 21}
{"loss": 0.5864171981811523, "acc": 0.6962997913360596, "length": 2.5, "mode": "test", "epoch": 21}
{"loss": 0.5567712187767029, "acc": 0.7144667506217957, "length": 2.404935121536255, "mode": "train", "epoch": 22}
{"loss": 0.5829087495803833, "acc": 0.698657214641571, "length": 2.546875, "mode": "test", "epoch": 22}
{"loss": 0.5513227581977844, "acc": 0.7206189036369324, "length": 2.539381742477417, "mode": "train", "epoch": 23}
{"loss": 0.5831880569458008, "acc": 0.7006592154502869, "length": 2.546875, "mode": "test", "epoch": 23}
{"loss": 0.5443505048751831, "acc": 0.725419282913208, "length": 2.559990167617798, "mode": "train", "epoch": 24}
{"loss": 0.5800449252128601, "acc": 0.7097458243370056, "length": 2.6875, "mode": "test", "epoch": 24}
{"loss": 0.5406448841094971, "acc": 0.7263840436935425, "length": 2.682992458343506, "mode": "train", "epoch": 25}
{"loss": 0.5731358528137207, "acc": 0.7106475830078125, "length": 2.79296875, "mode": "test", "epoch": 25}
{"loss": 0.5354381203651428, "acc": 0.732746422290802, "length": 2.8616292476654053, "mode": "train", "epoch": 26}
{"loss": 0.5743468999862671, "acc": 0.7110260725021362, "length": 2.93359375, "mode": "test", "epoch": 26}
{"loss": 0.528523325920105, "acc": 0.7382969260215759, "length": 2.9227688312530518, "mode": "train", "epoch": 27}
{"loss": 0.5850460529327393, "acc": 0.7085357308387756, "length": 3.12109375, "mode": "test", "epoch": 27}
{"loss": 0.5183342099189758, "acc": 0.7461125254631042, "length": 3.0081610679626465, "mode": "train", "epoch": 28}
{"loss": 0.5758512020111084, "acc": 0.7112136483192444, "length": 3.02734375, "mode": "test", "epoch": 28}
{"loss": 0.5170864462852478, "acc": 0.7498759627342224, "length": 2.927863121032715, "mode": "train", "epoch": 29}
{"loss": 0.5785123109817505, "acc": 0.7123657464981079, "length": 2.96875, "mode": "test", "epoch": 29}
{"loss": 0.5090799927711487, "acc": 0.7501087188720703, "length": 3.007978916168213, "mode": "train", "epoch": 30}
{"loss": 0.5800944566726685, "acc": 0.7182433605194092, "length": 3.02734375, "mode": "test", "epoch": 30}
{"loss": 0.5097746253013611, "acc": 0.7530338764190674, "length": 3.0544064044952393, "mode": "train", "epoch": 31}
{"loss": 0.5765581130981445, "acc": 0.7160583138465881, "length": 3.1796875, "mode": "test", "epoch": 31}
{"loss": 0.5045057535171509, "acc": 0.7562854290008545, "length": 3.0802981853485107, "mode": "train", "epoch": 32}
{"loss": 0.5672862529754639, "acc": 0.7252944707870483, "length": 3.14453125, "mode": "test", "epoch": 32}
{"loss": 0.49816182255744934, "acc": 0.7573041915893555, "length": 3.1237285137176514, "mode": "train", "epoch": 33}
{"loss": 0.5668354034423828, "acc": 0.7227126359939575, "length": 3.08984375, "mode": "test", "epoch": 33}
{"loss": 0.49435344338417053, "acc": 0.7600846886634827, "length": 3.1449568271636963, "mode": "train", "epoch": 34}
{"loss": 0.5882472991943359, "acc": 0.7251983284950256, "length": 3.234375, "mode": "test", "epoch": 34}
{"loss": 0.4866475760936737, "acc": 0.7635406255722046, "length": 3.284569025039673, "mode": "train", "epoch": 35}
{"loss": 0.5555557012557983, "acc": 0.7322723865509033, "length": 3.37109375, "mode": "test", "epoch": 35}
{"loss": 0.47994348406791687, "acc": 0.768047571182251, "length": 3.4198546409606934, "mode": "train", "epoch": 36}
{"loss": 0.5615009069442749, "acc": 0.7298674583435059, "length": 3.61328125, "mode": "test", "epoch": 36}
{"loss": 0.4785250127315521, "acc": 0.7670802474021912, "length": 3.571183919906616, "mode": "train", "epoch": 37}
{"loss": 0.5652861595153809, "acc": 0.7270904779434204, "length": 3.69921875, "mode": "test", "epoch": 37}
{"loss": 0.47239282727241516, "acc": 0.7707529067993164, "length": 3.6802327632904053, "mode": "train", "epoch": 38}
{"loss": 0.5594385266304016, "acc": 0.731768786907196, "length": 3.6953125, "mode": "test", "epoch": 38}
{"loss": 0.4667901396751404, "acc": 0.772324800491333, "length": 3.6585443019866943, "mode": "train", "epoch": 39}
{"loss": 0.5652960538864136, "acc": 0.7327361702919006, "length": 3.8359375, "mode": "test", "epoch": 39}
{"loss": 0.457864910364151, "acc": 0.7764873504638672, "length": 3.748405933380127, "mode": "train", "epoch": 40}
{"loss": 0.5628758668899536, "acc": 0.7329421043395996, "length": 3.7578125, "mode": "test", "epoch": 40}
{"loss": 0.46028566360473633, "acc": 0.7765759229660034, "length": 3.7134904861450195, "mode": "train", "epoch": 41}
{"loss": 0.5545924305915833, "acc": 0.7293579578399658, "length": 3.83984375, "mode": "test", "epoch": 41}
{"loss": 0.45522812008857727, "acc": 0.7785797715187073, "length": 3.7661821842193604, "mode": "train", "epoch": 42}
{"loss": 0.5677822828292847, "acc": 0.7301925420761108, "length": 3.73828125, "mode": "test", "epoch": 42}
{"loss": 0.4541183412075043, "acc": 0.7791422605514526, "length": 3.711805820465088, "mode": "train", "epoch": 43}
{"loss": 0.5527485609054565, "acc": 0.7373732924461365, "length": 3.83984375, "mode": "test", "epoch": 43}
{"loss": 0.44605231285095215, "acc": 0.7874091267585754, "length": 3.7945075035095215, "mode": "train", "epoch": 44}
{"loss": 0.5585972666740417, "acc": 0.7411011457443237, "length": 3.8984375, "mode": "test", "epoch": 44}
{"loss": 0.44338470697402954, "acc": 0.7863994240760803, "length": 3.8224399089813232, "mode": "train", "epoch": 45}
{"loss": 0.5491169691085815, "acc": 0.7477782964706421, "length": 3.9609375, "mode": "test", "epoch": 45}
{"loss": 0.435994952917099, "acc": 0.7882982492446899, "length": 3.8777005672454834, "mode": "train", "epoch": 46}
{"loss": 0.53223717212677, "acc": 0.748577892780304, "length": 3.953125, "mode": "test", "epoch": 46}
{"loss": 0.43652138113975525, "acc": 0.7890589237213135, "length": 3.844871997833252, "mode": "train", "epoch": 47}
{"loss": 0.5330545902252197, "acc": 0.7473755478858948, "length": 3.953125, "mode": "test", "epoch": 47}
{"loss": 0.4279337227344513, "acc": 0.7942898273468018, "length": 3.87111496925354, "mode": "train", "epoch": 48}
{"loss": 0.5323412418365479, "acc": 0.7514129281044006, "length": 3.9765625, "mode": "test", "epoch": 48}
{"loss": 0.42083391547203064, "acc": 0.7994639873504639, "length": 3.889901876449585, "mode": "train", "epoch": 49}
{"loss": 0.5335315465927124, "acc": 0.7556228041648865, "length": 3.9765625, "mode": "test", "epoch": 49}
{"loss": 0.41878435015678406, "acc": 0.8029460310935974, "length": 3.92856764793396, "mode": "train", "epoch": 50}
{"loss": 0.5246378183364868, "acc": 0.7570175528526306, "length": 3.9921875, "mode": "test", "epoch": 50}
{"loss": 0.41418519616127014, "acc": 0.8039582371711731, "length": 3.9321963787078857, "mode": "train", "epoch": 51}
{"loss": 0.5182065963745117, "acc": 0.7572570443153381, "length": 3.96875, "mode": "test", "epoch": 51}
{"loss": 0.4073028564453125, "acc": 0.8069838881492615, "length": 3.9076008796691895, "mode": "train", "epoch": 52}
{"loss": 0.5222072601318359, "acc": 0.7500548958778381, "length": 3.9765625, "mode": "test", "epoch": 52}
{"loss": 0.4091898500919342, "acc": 0.8091964721679688, "length": 3.93261456489563, "mode": "train", "epoch": 53}
{"loss": 0.4976004362106323, "acc": 0.7643981575965881, "length": 3.9921875, "mode": "test", "epoch": 53}
{"loss": 0.40335747599601746, "acc": 0.8081509470939636, "length": 3.936326503753662, "mode": "train", "epoch": 54}
{"loss": 0.5174975395202637, "acc": 0.7544876337051392, "length": 3.9921875, "mode": "test", "epoch": 54}
{"loss": 0.3968674838542938, "acc": 0.8133601546287537, "length": 3.932875156402588, "mode": "train", "epoch": 55}
{"loss": 0.5257585048675537, "acc": 0.7530959844589233, "length": 3.97265625, "mode": "test", "epoch": 55}
{"loss": 0.39255258440971375, "acc": 0.8134579062461853, "length": 3.929399251937866, "mode": "train", "epoch": 56}
{"loss": 0.514087975025177, "acc": 0.7598816156387329, "length": 3.9921875, "mode": "test", "epoch": 56}
{"loss": 0.3909332752227783, "acc": 0.8129357099533081, "length": 3.9455223083496094, "mode": "train", "epoch": 57}
{"loss": 0.5118532180786133, "acc": 0.7594481706619263, "length": 3.9765625, "mode": "test", "epoch": 57}
{"loss": 0.38822588324546814, "acc": 0.8179004192352295, "length": 3.932295083999634, "mode": "train", "epoch": 58}
{"loss": 0.4930485486984253, "acc": 0.7636261582374573, "length": 3.98828125, "mode": "test", "epoch": 58}
{"loss": 0.3841243386268616, "acc": 0.81920325756073, "length": 3.9434056282043457, "mode": "train", "epoch": 59}
{"loss": 0.49218323826789856, "acc": 0.7688506245613098, "length": 3.984375, "mode": "test", "epoch": 59}
{"loss": 0.37937378883361816, "acc": 0.8210347890853882, "length": 3.939980983734131, "mode": "train", "epoch": 60}
{"loss": 0.48925477266311646, "acc": 0.7698073983192444, "length": 3.98828125, "mode": "test", "epoch": 60}
test accuracy: 0.716714084148407
